docs/README.rst

```diff
@@ -25,44 +25,110 @@ Development documentation preview
+| on a page.                           | Local Editor     | generate a preview before creating the PR.      |
++--------------------------------------+------------------+-------------------------------------------------+
+
+Finding source content to edit
+------------------------------
+
+Sphinx has _roles_ and _directives_, where Markdown docs frameworks often do not have similar functionality. This means that Airflow uses directives
+to pull code examples, autogenerate indexes and tables of contents, and reference from resources across the codebase and across documentation provider packages.
+This can make it confusing to find the content source on certain types of reference pages.
+
+For example, in `Command Line Interface and Environment Variables Reference <https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#environment-variables>`__, the CLI reference is `autogenerated <https://github.com/apache/airflow/blob/main/docs/apache-airflow/cli-and-env-variables-ref.rst?plain=1#L44>`__,
+and requires more complex scripting. While the `Environment Variables <https://github.com/apache/airflow/blob/main/docs/apache-airflow/cli-and-env-variables-ref.rst?plain=1#L51>`__ are explicitly written.
+
+To make an edit to an autogenerated doc, you need to make changes to a string in the Python source file. In the previous example, to make edits to the CLI command reference text, you must edit the `cli_config.py <https://github.com/apache/airflow/blob/main/airflow/cli/cli_config.py#L1861>`__ source file.
+
 Building documentation
 ======================
 
-To generate a local version you can use `<../BREEZE.rst>`_.
+To generate a local version of the docs you can use `<../BREEZE.rst>`_.
 
 The documentation build consists of verifying consistency of documentation and two steps:
 
 * spell checking
 * building documentation
 
-You can only run one of the steps via ``--spellcheck-only`` or ``--docs-only``.
+You can choose to run the complete build, to build all the docs and run spellcheck. Or, you can use package names and the optional flags, ``--spellcheck-only`` or ``--docs-only`` to choose the scope of the build.
+
+Build all docs and spell check them:
 
 .. code-block:: bash
 
     breeze build-docs
 
-or just to run spell-check
+Just run spellcheck:
 
 .. code-block:: bash
 
      breeze build-docs --spellcheck-only
 
-or just to run documentation building
+Build docs without checking spelling:
 
 .. code-block:: bash
 
      breeze build-docs --docs-only
 
-Also, you can only build one documentation via ``--package-filter``.
+Build documentation of just one provider package:
 
 .. code-block:: bash
 
-    breeze build-docs --package-filter <PACKAGE-NAME>
+    breeze build-docs PACKAGE_ID
+
+Or, build docs for more than one provider package in the same command:
+
+.. code-block:: bash
+
+    breeze build-docs PACKAGE1_ID PACKAGE2_ID
```

Comment:
Should this be an example with a real value (e.g. `breeze build-docs apache.beam`)?

Coming from the "old school" version of this (i.e. being used to package name, `apache-airflow-providers-apache-beam` as the analogous example), I would find it useful to know what exactly a "package ID" is here.